doc/METADATA.ini

```diff
@@ -679,6 +679,43 @@ type= enum
 	RFC2822
 description= Support of some standards for a date specification.
 
+[check/permission]
+status= idea
+usedby/plugin= path
+type= enum
+description= A check to see if a certain user has correct permissions on a file. Permissions are
+	either r (read) w (write) x (execute) and any combination of them.
+	 The respective user which must have these permissions is declared in check/permission/user.
+example = "rwx ; rw ; r ; x ; w ; wx"
+
+[check/permission/user]
+status= idea
+usedby/plugin= path
+type= string
+	empty
+description= Used be check/permission to see if the given user has the correct permissions. If empty
+	it takes the current user.
+
+[check/port]
+status = idea
+usedby/plugin = network
+type = empty
+desciption = If specified, checks if the value of the key is between 0 and 65535.
```

Comment:
As non-root it is 1025-65535. Without checking this it would make little sense to have a separate check? (check/type could be used).

And the translation from port name -> number should also be done (done by getaddrinfo anyway, see below)

Reply:
What about emitting a warning when trying to set a port below 1025? I think the usability would suffer if users would need to add a separate check just to be assured that a port below 1025 is executed as root.

Also how exactly would you think of check/type? Can you give an example?

Reply:
You are right, the root check only makes sense for servers, not for clients. So there are 3 ways:

1. If a user simply wants a check from 0-65535, she would use `check/type=unsigned_short`.
2. If a user wants to check if a string can be used as port (number 0-65535 or a name from /etc/services), she would use check/port
3. If a user wants to check if a string can be used as port to listen to (as above + not occupied + root for 0-1024), she would use check/port/listen

Btw. is 0 a valid port number?